source/adapters/level_zero/command_buffer.cpp

```diff
@@ -41,17 +41,30 @@ bool IsDriverVersionNewerOrSimilar(ur_context_handle_t Context,
           (DriverVersionMinor >= VersionMinor) &&
           (DriverVersionBuild >= VersionBuild));
 }
+
+// Default to using compute engine for fill operation, but allow to
+// override this with an environment variable.
+static bool PreferCopyEngineForFill = [] {
```

Comment:
Don't think `static` adds anything here when we're in an anonymous namespace.

Reply:
Agreed, have removed `static` here.